protocol/statetracker/updaters/pairing_updater.go

```diff
@@ -128,6 +147,10 @@ func (pu *PairingUpdater) updateInner(latestBlock int64) {
 				continue
 			}
 		}
+
+		for _, updatable := range pu.pairingStakeEntriesUpdatables {
```

Comment:
instead of adding a new updater type, we can just update the stake from inside the consumer session manager for a specific chain. also makes more sense to have it there after filtering.

Reply:
Fixed in 437c61458270cbd7bbf5f4246e0f76407bec3139